sample.go

```
@@ -0,0 +1,21 @@
+package main
+
+import "net/http"
+
+var data = []byte{
+}
+
+type User struct {
+}
+
+func sample() []byte {
+	var method string
+	if method == http.MethodGet {
+		return data
+	}
+	return nil
+}
+
+func (u *User) BillingAddress() string {
```

Returning an empty string here may hide errors. What do you think about returning `(string, error)`?